context/brand-context/springernature/scss/enhanced.scss

```diff
@@ -11,4 +11,8 @@
 
 // 40-base
 @import '40-base/layout';
-@import '40-base/links';
\ No newline at end of file
+@import '40-base/links';
+@import '40-base/typography';
+
+// 60-utilities
+@import ' 60-utilities/typography';
```

Comment:
don't include the utilities level in the context output - these get included by the product using the context and appear after everything else (so they can override component styles)

Reply:
gotcha